zerver/tests/test_reactions.py

```diff
@@ -98,14 +98,14 @@ def test_emoji_name_to_emoji_code(self) -> None:
         An emoji name is mapped canonically to emoji code.
         """
         realm = get_realm('zulip')
+        emoji = RealmEmoji.objects.get(name="green_tick")
```

Comment:
I'd call this `realm_emoji`, or `green_tick`, for clarity.

Reply:
Done!